README.md

```diff
@@ -2,6 +2,9 @@
 
 # This repo is for development, there may be bugs and new features
 
+# Notice
+- To use the new LDSR settings please make sure to re-clone the LDSR repo.

```

Comment:
Please expand a little bit on this?

Reply:
The idea is to clone the LDSR repo again to get the new LDSR module that takes the arguments (instruction are a bit further down with the other upscalers), how would yo suggest rewriting it?